pkg/devcontainer/image_manager.go

```diff
@@ -0,0 +1,147 @@
+package devcontainer
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/artmoskvin/hide/pkg/util"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/registry"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/rs/zerolog/log"
+)
+
+type ImageManager interface {
```

Comment:
Let's use [this](https://github.com/moby/moby/blob/master/client/interface.go#L89)

Reply:
Done